src/js/mdjson-linter.js

```diff
@@ -0,0 +1,93 @@
+/*
+ *
+ * Copyright (c) 2017 Raising the Floor International.
+ *
+ * Licensed under the BSD-3-Clause license.
+ *
+ */
+// Inspired by the approach used here: https://github.com/matsu-chara/experimental-markdown-json-lint
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+var mdParser = require("markdown-to-ast");
+var JSON5    = require("json5");
+
+fluid.registerNamespace("gpii.mdjsonLinter");
+
+/**
+ *
+ * @param markdownString {String} - The markdown content to be parsed.
+ * @returns {Array} An array containing any errors detected.
```

Comment:
I think we use @return rather than @returns

Reply:
Fixed.